utils/generate_html_pages.sh

```diff
@@ -16,33 +16,35 @@ fi
 
 PAGES_DIR=$1
 
-mkdir $PAGES_DIR
+mkdir -p $PAGES_DIR
 STATS_DIR=$PAGES_DIR/statistics
-mkdir $STATS_DIR
+mkdir -p $STATS_DIR
 touch $STATS_DIR/index.html
 echo "<html>" > $STATS_DIR/index.html
```

Comment:
```suggestion
echo "<!DOCTYPE html>" > index.html
echo '<html lang="en">' >> index.html
echo '<meta charset="utf-8" />' >>
```
If we want this to be a valid HTML5 document.